experimental/ExperimentalTemplate/src/ExperimentalTemplate.jl

```diff
@@ -0,0 +1,15 @@
+# Add your new types, functions, and methods here.
+
+mutable struct ExampleStruct
+  i::Int
+end
+
+@doc raw"""
+    content(S::ExampleStruct)
+
+This is a dummy sample function to teach the use of docstrings.
+"""
+function content(S::ExampleStruct)
```

Comment:
There is a genuine `Oscar.content` function and now this docstring will show up in its docstring. Not good.

I would suggest (a) a different name for this dummy function, and (b) perhaps put this entire file into a `module` to avoid further clashes

Reply:
Could we maybe just ignore the template when loading Oscar by putting it into the `oldexppkgs` in `src/Oscar.jl`? I don't really see any benefit for maintenance when we load this. Only very minor one.

Reply:
I thought a bit and I think it makes sense to have both: A template that people with existing code can use for adjusting their stuff, as well as a function that copies the template and renames appropriately if someone wants to start from scratch. I'll do the second part once this PR is merged.